README.md

```diff
@@ -96,6 +96,8 @@ library for it project:
   enabled on buffers.
 * **WITH_DCACHE_RSC_TABLE** (default OFF): Build with data cache operations
   enabled on resource table.
+* **WITH_DCACHE** (default OFF): Build with all cache operations
```

Comment:
This commit can be merge with the previous one

Reply:
done